Funky-Chickens/bangazon-ecommerce
# fix migration order and seed files to get database populated

We made the following changes:
- fix migration order and seed files to get database populated with fake data
- change associations for join table to include a through statement for join table
- change seeder data to work with migrations

Reason:  The database migration would work before, but the data couldn't be seeded into the database because of the dependency problem.  Now it works as intended.

To test: 
- Run ```sequelize db:migrate``` and ```sequelize db:seed:all```.
- Open your terminal and go into psql.
- Connect to your database with ``` \c databasename```.
- Check to see if the tables are there with ```\d```.
- Check the data in each table with a select statement ```SELECT * FROM "Tablename";``` (Don't forget your semicolons....)  :)

Refer to issue #28
